README.md

```diff
@@ -1,84 +1,129 @@
-- we should create equivalent routes (and their associated controllers) inside the project which will use the flow-builder
+### How to work on the flow-builder repo, and consume from an other app (consumer app)?
+#### Steps on the community flow-builder repo:
+- create the feature branch
+- commit changes
+- run `yarn build-bundle` to build the app for the consumer app
+- commit changed files
+- push
 
-### Changes on JS dependencies
-#### /src/lib/mixins/FlowUploader.js
+#### Steps on the consumer app
+Let's assume we already have added the `"@floip/flow-builder"` as part of our dependencies. That means, we have something like this in our packages.json
+```
+"dependencies": {
+    ...
+    "@floip/flow-builder": "https://github.com/FLOIP/flow-builder.git#master",
+    ...
+}
+```
+We just need to:
+- create
+- go to the packages.json
+- create a feature branch (recommended to use the same name as in the flow-builder community), eg: 'feature/my-branch'
+- update the `@floip/flow-builder` by changing this `"@floip/flow-builder": "https://github.com/FLOIP/flow-builder.git#master"` into `"@floip/flow-builder": "https://github.com/FLOIP/flow-builder.git#feature/my-branch"`
+- run `yarn install`. Note that this will update the `yarn.lock`
+- then serve the app as defined in the packages.json (eg: `yarn watch`)
+
+If we made a change on the community flow-builder branch and our `packages.json` is already referring to that branch, then we just need to update it:
+- run `yarn up -i @floip/flow-builder`
+- hit enter to choose the 1st option
+- then serve the app
+
+### How to consume changes from flow-runner's feature branch?
+Just imitate the above section "How to work on the flow-builder repo, and consume from an other app (consumer app)?", by:
+- changing the dependency repo to @floip/flow-runner
+- changing the consumer repo to floip/flow-builder
+
+### Changes on JS dependencies: /src/lib/mixins/FlowUploader.js
 This is related to `Flow.js` library which is providing multiple simultaneous, stable and resumable uploads via the HTML5 File API.
 - On standalone UI, we import the Flow like `import Flow from '@flowjs/flow.js';` and use it inside the file like `new Flow(...)`
 - But when we integrate the flow-builder into other project, we may need to call a global Flow like `new global.Flow(...)`.
 
-## Usage
+### Storybook
+1. You only need to serve it with `yarn storybook:serve`
+2. Then follow the provided url, by default it should be `http://localhost:6006/` (but in case the port `6006` is non-available, it may use different one)
 
-### Create new flow
+### An explanation of app.config.json vs builder.config.json
+`app.config.json` is intended to support the configuration needed when integrating this flow building ui into a larger platform or application while `builder.config.json` is intended to configure those features which are core to the flow builder - the blocks enabled, languages or content types supported etc.
```

Comment:
If you know about `app.config.json` and can add a bit more about it - maybe examples of what we are configuring - that would be nice. I still don't understand what we need it for 😄